docs/DeveloperGuide.adoc

```diff
@@ -294,18 +298,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As an ... |I want to ... |So that I can...
+|`* * *` |Administrator |check on all of/ sort the deliverymen at once |better manage the deliverymen
+
+|`* * *` |Administrator |help customers add/edit/remove orders |better manage customers' orders
```

Comment:
Would be good to separate the different commands into different use cases.
eg. "help customers add orders" is one user story